pkgs/development/python-modules/wxPython/4.0.nix

```
@@ -22,14 +37,43 @@ buildPythonPackage rec {
     sha256 = "35cc8ae9dd5246e2c9861bb796026bbcb9fb083e4d49650f776622171ecdab37";
   };
 
-  nativeBuildInputs = [
-    pkgconfig
-  ];
+  doCheck = false;
 
-  buildInputs = [
-    gtk3 libjpeg libtiff SDL gst-plugins-base libnotify freeglut xorg.libSM
-    which
-  ];
+  nativeBuildInputs = [ pkgconfig which doxygen wxGTK libX11 ];
+
+  buildInputs = with python3Packages; [ libjpeg libtiff SDL
+      gst-plugins-base libnotify freeglut xorg.libSM ncurses
+      requests libpng gstreamer
+      pathlib2
```

it's a python2 only library
```suggestion
      lib.optional isPy27 pathlib2
```

---

surprisingly, this is needed on python3, too.